docs/product-getting-started-guide/src/main/asciidoc/decision-table-create-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='dec-table-create-proc']
+= Creating a speed violation guided decision table
+
+The following chapter instructs you how to create a speed violation decision table in {PRODUCT} using the Guided Decision Table wizard.
```

Comment:
See my comment in the Creating the Violation data object data fields section, about intro statements.

Reply:
done